packages/stacked/example/lib/ui/builder_widget_example/builder_widget_example_view.dart

```diff
@@ -2,25 +2,21 @@ import 'package:flutter/material.dart';
 import 'package:example/ui/home/home_viewmodel.dart';
 import 'package:stacked/stacked.dart';
 
-class BuilderWidgetExampleView extends ViewModelBuilderWidget<HomeViewModel> {
+class BuilderWidgetExampleView extends StackedView<HomeViewModel> {
```

Comment:
Nice 🚀 We should note this down as a breaking change please. 

It needs to go into our migration section.